content/javascript/concepts/storage/terms/clear.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: '.clear()'
+Description: '.clear() removes all keys in a Storage object.'
+Subjects:
+  - 'Data Visualization'
+  - 'Web Development'
+Tags:
+  - 'APIs'
+  - 'Browsers'
+  - 'Data Types'
+  - 'Methods'
+  - 'Tuples'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**.clear()** removes all keys in a Storage object.
+
+## Syntax
+```pseudo
+Storage.clear()
+// return: undefined
+```
+The method does not accept arguments. It will clear all storage keys and their respective values.
+
+## Codebyte Example
```

Comment:
```suggestion
## Example
```

Reply:
I was under the impression that codebyte examples required the "## Codebyte Example" header and did not require and "output" block? Are codebyte examples only used when a regular Example block is already present?

Reply:
The problem here is that the code as written doesn't work with the engine running the codebytes.

Reply:
Is there a way to check what the md file looks like once it goes online? I don't know how to display markdown files on my local machine without seeing them in github. Are there any programs that support markdown?

Reply:
> Is there a way to check what the md file looks like once it goes online? I don't know how to display markdown files on my local machine without seeing them in github. Are there any programs that support markdown?

@MB1721, there's a list here: https://becomeawritertoday.com/best-markdown-editor/